elasticsearch/examples/networkpolicy/Makefile

```diff
@@ -0,0 +1,16 @@
+default: test
+include ../../../helpers/examples.mk
+
+RELEASE := helm-es-networkpolicy
+
+install:
+	helm upgrade --wait --timeout=600s --install $(RELEASE) --values ./values.yaml ../../ ; \
+
+restart:
+	helm upgrade --set terminationGracePeriod=121 --wait --timeout=600s --install $(RELEASE) --values ./values.yaml ../../ ; \
+
+test: install
+	helm test $(RELEASE)
+
```

Comment:
>Should it be something like
>```
>test: install
>	helm test $(RELEASE)
>```

I would suggest dropping completely the test target if there is no specific tests:

```suggestion
```